src/csv_to_engagement_db/configuration.py

```diff
@@ -1,12 +1,88 @@
+from datetime import datetime
+
```

Comment:
```suggestion
```

Reply:
I've removed this but note that pycharm's optimise imports inserts newlines between different types of import automatically, so we'd need to add an exception to the auto-formatting rules and run a global optimise imports if we wanted to do this everywhere